welcome/src/main/kotlin/edu/artic/welcome/ToursAdapter.kt

```diff
@@ -0,0 +1,56 @@
+package edu.artic.welcome
+
+import android.content.Context
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import com.bumptech.glide.Glide
+import edu.artic.db.models.ArticTour
+import edu.artic.welcome.R.id.stops
+
+/**
+ * @author Sameer Dhakal (Fuzz)
+ */
+
+class ToursAdapter(val tours: List<ArticTour>, private val context: Context) : RecyclerView.Adapter<ToursViewHolder>() {
```

Comment:
Take a look at the `BaseRecyclerViewAdapter`. I can show you today. It does all the work for you.